internal/chart/chart_cmd.go

```diff
@@ -117,7 +116,7 @@ func chartCmdExecute(t api.Builder, tracer trace.Tracer) func(cmd *cobra.Command
 		if err != nil {
 			return fmt.Errorf("creting file: %w", err)
 		}
-		defer errorh.SafeClose(f)
```

Comment:
Doesn't this logging make sense? If we fail to close the file, doesn't that imply we might not have flushed the content properly? if we remove the log we wouldn't have any observability of when it goes wrong any more. 

Alternatively, instead of deferring `f.Close()`, we could just finish with it and return, that way the error bubbles up without having logging.